Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2367 @@
+    """
+    # Default payload
+    payload: dict[str, Any] = {'verb': 'query'}
+    page_size = args.get('page_size')
+    max_limit = args.get('limit', DEFAULT_PAGE_SIZE)
+
+    if page_size:
+        if page_limit >= max_limit:
+            # in case user pass the page_size or limit is less than page_size
+            payload['limit'] = page_limit
+    else:
+        payload['limit'] = page_limit if page_limit != DEFAULT_PAGE_SIZE else max_limit
+
+    from_time = iso_creation_date(args.get('start_time', ''))
+    to_time = iso_creation_date(args.get('end_time', ''))
+
+    if from_time:
+        payload['start_time'] = from_time
+
+    if to_time:
+        payload['end_time'] = to_time
+
+    return payload
+
+
+def get_params_query(args: dict, page_limit: int = 0) -> dict:
+    """
+    This function creates a query param based on the demisto.args().
+    Args:
+        args: demisto.args()
+        page_limit: Page Limit (int)
+    Returns:
+        Return arguments dict.
+    """
+    query_param: dict = {}
+    ip = args.get('ip')
+    url = args.get('url')
+    md5 = args.get('md5')
+    sha256 = args.get('sha256')
+
+    if ip:
+        check_valid_indicator_value('ip', ip)
+
+    if md5:
+        check_valid_indicator_value('md5', md5)
+
+    if sha256:
+        check_valid_indicator_value('sha256', sha256)
+
+    max_limit = args.get('limit', DEFAULT_PAGE_SIZE)
+    page_size = args.get('page_size')
+
+    if page_size and (page_limit > max_limit):
+        # in case user pass the page_size or limit, limit will ignore
+        query_param['limit'] = page_limit
+    else:
+        query_param['limit'] = page_limit if page_limit != DEFAULT_PAGE_SIZE else max_limit
+
+    query_param['ip'] = ip
+    query_param['url'] = url
+    query_param['sha256'] = sha256
+    query_param['id'] = arg_to_number(args.get('allowlist_id'), arg_name='allowlist_id')
+    query_param['domain'] = args.get('domain')
+
+    return query_param
+
+
+def check_valid_indicator_value(indicator_type: str, indicator_value: str) -> bool:
+    """
+    Check the validity of indicator values
+    Args:
+        indicator_type: Indicator type provided in the command
+            Possible Indicator type are : sha256, urls, ip, md5
+        indicator_value: Indicator value provided in the command
+    Returns:
+        True if the provided indicator values are valid
+    """
+    if indicator_type == 'ip':
+        if not is_ip_valid(indicator_value):
+            raise ValueError(f'"{indicator_value}" is not a valid IP')
+        return True
+
+    hash_to_regex: dict[str, Any] = {
+        'sha256': sha256Regex,
+        'urls': urlRegex,
+        'md5': md5Regex
+    }
+    if indicator_type in hash_to_regex:
+        if not re.match(hash_to_regex[indicator_type], indicator_value):
+            raise ValueError(f'{indicator_value} is not a valid {indicator_type}')
+    else:
+        raise ValueError(f'Indicator {indicator_type} type does not support')
```

Comment:
```suggestion
        raise ValueError(f'Indicator {indicator_type} type is not supported. Supported types: {",".join(hash_to_regex.kers()}')
```

Please check if it works, I'm not sure about the syntax (or do it any other way)